src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -52,8 +52,9 @@ namespace NuGetVSExtension
         Window = "{34E76E81-EE4A-11D0-AE2E-00A0C90FFFC3}", // this is the guid of the Output tool window, which is present in both VS and VWD
         Orientation = ToolWindowOrientation.Right)]
     [ProvideOptionPage(typeof(GeneralOptionPage), "NuGet Package Manager", "General", 113, 115, true)]
-    [ProvideOptionPage(typeof(PackageSourceOptionsPage), "NuGet Package Manager", "Package Sources", 113, 114, true, Sort = 0)]
-    [ProvideOptionPage(typeof(PackageSourceMappingOptionsPage), "NuGet Package Manager", "Package Source Mapping", 113, 116, true, Sort = 1)]
+    [ProvideOptionPage(typeof(ConfigPathsOptionsPage), "NuGet Package Manager", "Config Paths", 113, 117, true, Sort = 0)]
```

Comment:
The page name here doesn't match the screenshot. I'd expect the 3rd parameter to be "Configuration Files". Is it actually showing that currently? If it is I wonder where it's pulling it from 🤔